src/main/resources/application.yml

```diff
@@ -11,7 +11,7 @@ lotinformation_by_absent:
 lotinformation_by_date:
   initial_time: 18000000
 pdfparser:
-  initial_time: 240
+  initial_time: 24000000
```

Comment:
this is not supposed to be updated, is it?

Reply:
yes